necsim/impls/no-std/src/cogs/origin_sampler/almost_infinite/rectangle.rs

```diff
@@ -0,0 +1,105 @@
+use core::{
```

Comment:
This should just be merged with the wrapping noise origin sampler (perhaps even the non-spatial one by allowing demes beyond zero or one?)